lib/bigcommerce/index.ts

```diff
@@ -452,18 +466,17 @@ export async function getCollectionProducts({
   }
   const productList = res.body.data.site.category.products.edges.map((item) => item.node);
 
-  return bigcommerceToVercelProducts(productList);
+  return bigCommerceToVercelProducts(productList);
 }
 
 export async function getCollections(): Promise<VercelCollection[]> {
-  const res = await bigcommerceFetch<BigCommerceCollectionsOperation>({
+  const res = await bigCommerceFetch<BigCommerceCollectionsOperation>({
     query: getStoreCategoriesQuery
   });
   const collectionIdList = res.body.data.site.categoryTree.map(({ entityId }) => entityId);
-
   const collections = await Promise.all(
```

Comment:
Regarding fetching all categories via single query.  In theory, we could use categoryTree query for one request but [Vercel Collection](https://github.com/bigcommerce/nextjs-commerce/blob/main/lib/bigcommerce/types.ts#L31) requires SEO field that we can get via Category node.

Reply:
OK, I see the problem, let's leave it as is then.